File: Swift/Algorithms/Sorting/bubble_sort.swift
//Performs BubbleSort of an array passed as reference(inout)

func bubbleSort(arr: inout [Int]) {
    for i in stride(from: arr.count - 1, to: 1, by: -1) {
        for j in 0..<i where arr[j] > arr[j + 1] {
            swap(&arr[j], &arr[j + 1])
        }
    }
}

File: PHP/Algorithms/Sorting/BubbleSort.php
<?php

function bubble_sort($arr) {
    $length = count($arr)-1;
    
    for ($i=0; $i < $length; $i++) {
        for ($j=0; $j < $length-$i; $j++) {
            $k = $j+1;
            if ($arr[$k] < $arr[$j]) {
                list($arr[$j], $arr[$k]) = array($arr[$k], $arr[$j]);
            }
        }
    }

    return $arr;
}